NevisExampleApp/Domain/Use Case/DeleteAuthenticatorsUseCaseImpl.swift

```diff
@@ -42,7 +42,7 @@ extension DeleteAuthenticatorsUseCaseImpl: DeleteAuthenticatorsUseCase {
 		}
 
 		var responses: [Observable<OperationResponse>] = []
-		accounts.forEach { account in
+		for account in accounts {
```

Comment:
I think the SwiftFormat rule is missing here not to rewrite the `forEach`

Reply:
Fixed.